app/views/devise/mailer/invitation_instructions.html.erb

```diff
@@ -1,6 +1,10 @@
 <p><%= I18n.t("devise.mailer.invitation_instructions.hello", email: @resource.email) %></p>
```

Comment:
Let's not use the YML file and instead just write the text in the body. Start off by using the template from Close.io and modify it to fit our needs.